src/transformers/modeling_utils.py

```diff
@@ -781,46 +780,49 @@ def prune_heads(self, heads_to_prune: Dict[int, List[int]]):
 
         self.base_model._prune_heads(heads_to_prune)
 
-    def save_pretrained(self, save_directory: Union[str, os.PathLike]):
+    def save_pretrained(
+        self,
+        save_directory: Union[str, os.PathLike],
+        is_main: bool = True,
+        _save_function: Callable = torch.save,
```

Comment:
At the moment, it seems to me that `save_function` is an optional argument because `None` can be passed. I think, I'd prefer to have it non-optional so that a function always has to be passed.